libs/wingc/src/jsify.rs

```diff
@@ -440,7 +450,7 @@ impl JSifier {
 						module_name
 					}),
 					if module_name.name.starts_with("\"./") {
-						// TODO so many assumptions here, would only wort with a JS file
+						// TODO so many assumptions here, would only work with a JS file
```

Comment:
Tracking issue for this TODO?

(Let's try to make this a habit so we have better visibility of our tech debt and we can prioritize for beta)